auth0/v3/management/hooks.py

```diff
@@ -0,0 +1,188 @@
+                (depending on include_fields) from the result, empty to
+                retrieve all fields.
+
+            include_fields (bool, optional): True if the fields specified are
+                to be included in the result, False otherwise
+                (defaults to true).
+
+            page (int, optional): The result's page number (zero based).
+
+            per_page (int, optional): The amount of entries per page.
+
+            include_totals (bool, optional): True if the query summary is
+                to be included in the result, False otherwise.
+
+        See: https://auth0.com/docs/api/management/v2#!/Hooks/get_hooks
+        """
+
+        params = {
+            "fields": fields and ",".join(fields) or None,
+            "include_fields": str(include_fields).lower(),
+            "page": page,
+            "per_page": per_page,
+            "include_totals": str(include_totals).lower(),
+        }
+
+        # since the default is True, this is here to disable the filter
+        if enabled is not None:
+            params["enabled"] = str(enabled).lower()
+
+        return self.client.get(self._url(), params=params)
+
+    def create(self, body):
+        """Creates a new Hook.
+
+        Args:
+            body (dict): Attributes for the newly created hook,
+                See: https://auth0.com/docs/api/v2#!/Hooks/post_hooks
+        """
+        return self.client.post(self._url(), data=body)
+
+    def get(self, id, fields=None, include_fields=True):
+        """Retrieves a hook by its ID.
+
+        Args:
+            id (str): The id of the hook to retrieve.
+
+            fields (list, optional): A list of fields to include or exclude
+                (depending on include_fields) from the result, empty to
+                retrieve all fields.
+
+            include_fields (bool, optional): True if the fields specified are
+                to be included in the result, False otherwise
+                (defaults to true).
+
+        See: https://auth0.com/docs/api/management/v2#!/Hooks/get_hooks_by_id
+        """
+        params = {
+            "fields": fields and ",".join(fields) or None,
+            "include_fields": str(include_fields).lower(),
+        }
+        return self.client.get(self._url(id), params=params)
+
+    def delete(self, id):
+        """Deletes a hook.
+
+        Args:
+            id (str): The id of the hook to delete.
+
+        See: https://auth0.com/docs/api/management/v2#!/Hooks/delete_hooks_by_id
+        """
+        return self.client.delete(self._url(id))
+
+    def update(self, id, body):
+        """Updates an existing hook.
+
+        Args:
+            id (str): The id of the hook to modify.
+
+            body (dict): Attributes to modify.
+
+        See: https://auth0.com/docs/api/v2#!/Hooks/patch_hooks_by_id
+        """
+        return self.client.patch(self._url(id), data=body)
+
+    def get_secrets(self, id, fields=None, include_fields=True):
```

Comment:
Similarly to the above, this endpoint doesn't document these properties. Please remove fields and include_fields. 

https://auth0.com/docs/api/management/v2#!/Hooks/get_secrets

Reply:
I also tested that one and here the fields and include_fields parameter do not work. I'll push a fix to remove them. The mention "Accepts a list of fields to include or exclude in the result." should be removed from the docs.